theme/template.sublime-theme

```diff
@@ -0,0 +1,1026 @@
+		},
+		{
+			"class": "icon_folder_loading",
+			"layer0.texture":
+			{
+				"keyframes":
+				[
+					"Theme - Default/common/folder_loading_01.png",
+					"Theme - Default/common/folder_loading_02.png",
+					"Theme - Default/common/folder_loading_03.png",
+					"Theme - Default/common/folder_loading_04.png",
+					"Theme - Default/common/folder_loading_05.png",
+					"Theme - Default/common/folder_loading_06.png",
+					"Theme - Default/common/folder_loading_07.png",
+					"Theme - Default/common/folder_loading_08.png",
+					"Theme - Default/common/folder_loading_09.png",
+					"Theme - Default/common/folder_loading_10.png",
+					"Theme - Default/common/folder_loading_11.png",
+					"Theme - Default/common/folder_loading_12.png",
+					"Theme - Default/common/folder_loading_13.png",
+					"Theme - Default/common/folder_loading_14.png",
+					"Theme - Default/common/folder_loading_15.png",
+					"Theme - Default/common/folder_loading_16.png",
+					"Theme - Default/common/folder_loading_17.png",
+					"Theme - Default/common/folder_loading_18.png",
+					"Theme - Default/common/folder_loading_19.png",
+					"Theme - Default/common/folder_loading_20.png",
+				],
+				"loop": true,
+				"frame_time": 0.08,
+			},
+			"layer0.tint": "var(subtle)",
+			"layer0.opacity": "var(opaque)",
+			"content_margin": [9, 8]
+		},
+		{
+			"class": "icon_folder_dup",
+			"layer0.texture": "Theme - Default/common/symlink.png",
+			"layer0.tint": "var(subtle)",
+			"layer0.opacity": 1,
+			"content_margin": [9, 8]
+		},
+
+		// File Icons
+		{
+		  "class": "icon_file_type",
+		  "layer0.tint": "var(muted)",
+		  "layer0.opacity": 1,
+		  "content_margin": [8, 8]
+		},
+		// File Icons: Hovered
+		{
+		  "class": "icon_file_type",
+		  "parents": [{"class": "tree_row", "attributes": ["hover"]}],
+		  "layer0.tint": "var(muted)",
+		  "layer0.opacity": 1
+		},
+		// File Icons: Selected
+		{
+		  "class": "icon_file_type",
+		  "parents": [{"class": "tree_row", "attributes": ["selected"]}],
+		  "layer0.tint": "var(text)",
+		  "layer0.opacity": 1
+		},
+
+		// Tabs
+		{
+			"class": "tabset_control",
+			"layer0.texture": "",
+			"layer0.opacity": 1.0,
+			"layer0.tint": "var(surface)",
+			"layer0.inner_margin": 0,
+			"content_margin": [0, 4, 0, 4],
+			"tab_overlap": 1,
+			"tab_height": 26,
+			"connector_height": 0,
+			"spacing": 5,
+		},
+		{
+			"class": "tabset_control",
+			"settings": ["themed_title_bar"],
+			"content_margin": [0, 0, 0, 4],
+		},
+
+		{
+			"class": "tab_connector",
+			"layer0.texture": "",
+			"layer0.opacity": 1.0,
+			"tint_index": 0,
+		},
+
+		{
+			"class": "tab_control",
+			"layer0.texture": "",
+			"layer0.opacity": 1.0,
+			"layer0.tint": "var(surface)",
+			"layer0.inner_margin": 0,
+			"layer1.texture": "Theme - Default/common/button.png",
+			"layer1.opacity": 1.0,
+			"layer1.tint": "var(overlay)",
+			"layer1.inner_margin": 4,
+			"layer2.texture": "Theme - Rose Pine/images/tab_square_highlight.png",
```

Comment:
Should this be Rosé instead of Rose? If it works then ignore this suggestion.

```suggestion
			"layer2.texture": "Theme - Rosé Pine/images/tab_square_highlight.png",
```

Reply:
I tried that first, but ST doesn’t like the accent on the e, it messes up the file path and makes it so it can’t load the images. 🤷🏼‍♂️